tests/testthat/test-emmeans.R

```diff
@@ -45,3 +49,14 @@ test_that("ref.grid tidiers work", {
   check_tidy_output(td, strict = FALSE)
   check_dims(td, 36, 7)
 })
+
+test_that("summary_emm tidiers work", {
+  tdm <- tidy(marginal)
+  tdms <- tidy(marginal_summary)
+  
+  expect_identical(tdm, tdms)
+  
+  tdjt <- tidy(joint_tests_summary)
+  check_tidy_output(tdjt, strict = FALSE)
```

Comment:
Can you comment on why we need `strict = FALSE` here?

Reply:
Not sure why I added this. It's certainly not needed for the tests to pass. I'll remove it.